### ISSUE TITLE
v3.2.0 Release Plan

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-explorer-vscode#3286]
```
- Code Freeze: April 7, 2025
- RC Build: April 8, 2025
- Testing: April 8, 2025 - May 4, 2025
- GA: May 5, 2025
- System Demo: May 13, 2025
```

### Zowe Explorer

- https://github.com/zowe/zowe-explorer-vscode/issues/2386
- https://github.com/zowe/zowe-explorer-vscode/issues/2373

### Zowe Explorer API

- https://github.com/zowe/zowe-explorer-vscode/issues/3088
- https://github.com/zowe/zowe-explorer-vscode/issues/3346

### Bugs (could be in 3.1.x release)

- https://github.com/zowe/zowe-explorer-vscode/issues/3379
- https://github.com/zowe/zowe-explorer-vscode/issues/3372
- https://github.com/zowe/zowe-explorer-vscode/issues/3360
- https://github.com/zowe/zowe-explorer-vscode/issues/3345
- https://github.com/zowe/zowe-explorer-vscode/issues/3338
- https://github.com/zowe/zowe-explorer-vscode/issues/3337
- https://github.com/zowe/zowe-explorer-vscode/issues/3402
- https://github.com/zowe/zowe-explorer-vscode/issues/3330
- https://github.com/zowe/zowe-explorer-vscode/issues/3432
- https://github.com/zowe/zowe-explorer-vscode/issues/2294

### Project Docs

- https://github.com/zowe/zowe-explorer-vscode/issues/3367

### CICS 
- https://github.com/zowe/cics-for-zowe-client/issues/217
- https://github.com/zowe/cics-for-zowe-client/issues/18
- https://github.com/zowe/cics-for-zowe-client/issues/224
- https://github.com/zowe/cics-for-zowe-client/issues/56
- https://github.com/zowe/cics-for-zowe-client/issues/225

[2025 Zowe Explorer Roadmap](https://github.com/zowe/zowe-explorer-vscode/wiki/2025-Zowe-Explorer-Roadmap)

moved to 3.1.2:

- https://github.com/zowe/zowe-explorer-vscode/issues/3427

4/1/25 discussion with code freeze coming up.  
Items to be moved to 3.3.0, bugs and doc updates may release prior to 3.3.0.

- https://github.com/zowe/zowe-explorer-vscode/issues/1930
- https://github.com/zowe/zowe-explorer-vscode/issues/3171 - onHold
- https://github.com/zowe/zowe-explorer-vscode/issues/2939
- https://github.com/zowe/zowe-explorer-vscode/issues/2265
- https://github.com/zowe/zowe-explorer-vscode/issues/2978 - v2.18.2

Added repo-specific stories as sub-issues

**Comment by davenice** [comment]
For CICS
https://github.com/zowe/cics-for-zowe-client/issues/217
https://github.com/zowe/cics-for-zowe-client/issues/18 
https://github.com/zowe/cics-for-zowe-client/issues/224
https://github.com/zowe/cics-for-zowe-client/issues/56

Plus we're hoping to add some end to end testing - I don't have an issue open yet. Hopefully we can do more than this too 🤔

**Comment by JillieBeanSim** [comment]
@adam-wolfe will you be handling the 3.3.0 plan readout this go round?

**Comment by JillieBeanSim** [comment]
I can demo the Zowe Explorer: Update Profile Configuration Schema command, share importance of extenders including schema version for on disk schema updates and share new extender capability for team config creation.

**Comment by davenice** [comment]
> share new extender capability for team config creation.

@JillieBeanSim - do we share stuff that we haven't completed yet? For instance some of the work we want to do to improve the attributes view.

**Comment by JillieBeanSim** [comment]
> > share new extender capability for team config creation.
> 
> [@JillieBeanSim](https://github.com/JillieBeanSim) - do we share stuff that we haven't completed yet? For instance some of the work we want to do to improve the attributes view.

Hey @davenice the demo is for deliverable items that users can expect to see in the version demoed, sometimes we include important fixes from previous version patches that released after last demo.
If something still in progress and won't be published by demo day, can save until the next release demo to share.

Are you looking for feedback on the in progress work before publishing?

**Comment by davenice** [comment]
Ah ok - I thought "new extender capability for team config creation" sounded new, cool, and not done yet!

No - I don't think we necessarily need wider feedback on what we're doing, it's pretty CICS specific. There are a few questions around extensibility that it would be interesting to talk over with the Zowe Explorer team as a starter, but I think we can start there. :-)

**Comment by JillieBeanSim** [comment]
> Ah ok - I thought "new extender capability for team config creation" sounded new, cool, and not done yet!
> 
> No - I don't think we necessarily need wider feedback on what we're doing, it's pretty CICS specific. There are a few questions around extensibility that it would be interesting to talk over with the Zowe Explorer team as a starter, but I think we can start there. :-)

the `new extender capability for team config creation` is a new ZE API for extenders like the CICS that want to have a Create Team Config experience just like ZE offers users in our tree views. our process was moved to the ZE API package and even ZE calls the API now.

**Comment by davenice** [comment]
We could demo https://github.com/zowe/cics-for-zowe-client/issues/56 - we could also briefly describe what we did for the MFA work, if that's the kind of thing that happens.

**Comment by traeok** [comment]
I'd like to demo pagination for listing data sets and PDS members if that's alright 😋

**Comment by davenice** [comment]
I don't think we demoed the CICS extension at all last time, so perhaps we could (briefly!) talk over the visual changes we've made, the logging we've added, https://github.com/zowe/cics-for-zowe-client/issues/56, and mention the MFA work (which still needs a bit of work between us and base Zowe Explorer) - so a couple of minutes just quickly showing where we're up to.

**Comment by awharn** [comment]
I should be available to demo the data set search history, regex support, and case sensitivity support.

**Comment by JillieBeanSim** [comment]
See list above for items not completed and removed from 3.2.0 plan's sub-issues
